src/components/common/Stepper.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-container id="step-buttons-container">
     <template v-for="(step, index) in getSteps">
-      <div class="step" :key="index" @click="goTo(step)">
+      <div class="step" :key="index" @click="goTo(step)" v-on:keyup.tab="goTo(step)">
```

Comment:
Not sure if this was intended, but I was trying out the tabbing and it requires tabbing twice to get to the next step for me.

Reply:
I noticed that too actually, i'll see if there is a way to fix this.

Reply:
PS You can remove elements from the tab order by specifying `tabindex="-1"`.

Reply:
Thanks for the tip! Updated 👍

Reply:
Almost! looks like the check mark icon may also need a -1 tabIndex, when I try to tab through any of the steps that are valid it takes 2 tabs.

Reply:
Ahh!! haha thanks :)

Reply:
Hmm... weird. The first tabbing once clicked on the page requires two, but seems to cycle through the steps fine, whether valid or invalid. 🤔  Could you give me a sanity check and try again with a few of the steps valid?

Reply:
You are right, seems to be the first tab once clicked, other tabs work fine whether it is valid or not.

I think the main issue is resolved, if you think you know what is up go ahead and fix it, otherwise it doesn't seem to be a big problem. Resolve this if you are satisfied.

Reply:
Thanks sir! 

Seems like a universal thing, no matter where you click on the page the first tab just initializes and the second starts to cycle through the page. I didn't really notice until i cycled through every element on the page, when i got back to the stepper it cycled right through each step normally. 
Great observation though, thanks!

Reply:
Tabbing seems fine to me.